packages/core/src/utilities/createSigmoidRGBTransferFunction.ts

```diff
@@ -1,7 +1,7 @@
 import vtkColorTransferFunction from '@kitware/vtk.js/Rendering/Core/ColorTransferFunction';
 import vtkDataArray from '@kitware/vtk.js/Common/Core/DataArray';
 import { VOIRange } from '../types';
-import { windowLevel as windowLevelUtil } from '.';
+import * as windowLevelUtil from './windowLevel';
```

Comment:
I wonder how we can really identify these and lint error it

Reply:
If we just move to rollup it identifies these.  I found this one with rollup - plus 140 more, but only fixed one.